alerter.c

```diff
@@ -25,7 +25,9 @@ void alertInCelcius(float farenheit) {
 
 int main() {
     alertInCelcius(400.5);
+    assert(alertFailureCount==1);

```

Comment:
Atleast 3 test values need to be covered i.e. below threshold, threshold and above threshold.